### DISCUSSION TITLE
How to check train results on tensorboard?

### DISCUSSION THREAD

**Discussion by elcolex777** [voicepaw/so-vits-svc-fork#874] [Q&A]
Hello.
What do you say about my train results on tensorboard? I’m beginner and don’t understend how to check to stop train model. I have this results after 12 hours of training (see screenshot). I have model with 3280 epochs, but results of inferring not good((

![IMG_5046](https://github.com/voicepaw/so-vits-svc-fork/assets/140021099/b77f4549-f863-479a-aaa5-5050b4030d17)

I setup treining that:
`!svc pre-split
!svc pre-resample
!svc pre-config
!svc pre-hubert -fm crepe `

my config this:
`{
    "train": {
        "log_interval": 100,
        "eval_interval": 200,
        "seed": 1234,
        "epochs": 20000,
        "learning_rate": 0.0001,
        "betas": [
            0.8,
            0.99
        ],
        "eps": 1e-09,
        "batch_size": 16,
        "fp16_run": false,
        "bf16_run": false,
        "lr_decay": 0.999875,
        "segment_size": 10240,
        "init_lr_ratio": 1,
        "warmup_epochs": 0,
        "c_mel": 45,
        "c_kl": 1.0,
        "use_sr": true,
        "max_speclen": 512,
        "port": "8001",
        "keep_ckpts": 3,
        "num_workers": 4,
        "log_version": 0,
        "ckpt_name_by_step": false,
        "accumulate_grad_batches": 1
    }, `

do I need more training? I think next session with 12 hours give me model approx 7000 epochs. But I don’t know will be results better. Or may be I do something wrong? Please help. Thanks.
